src/tests/validity/variable_ref_operators.ab

```diff
@@ -0,0 +1,16 @@
+// Output
+// 0
+// 0
+
+fun value(value) {
+    let var = value
+    var += 1
+    var -= 1
+    var *= 1
+    var /= 1
+    var %= 1
+    echo var
+}
+
+value(4)
+value(-1)
```

Comment:
If you want this file to test operators of `ref` (references) then you should:
1. Pass to it a variable (to accept the reference)
2. Accept the reference in the function argument
3. Remove the variable assign in the body (we want to test references!)

Here are changes that will make it work:
```suggestion
fun value(ref value) {
    var += 1
    var -= 1
    var *= 1
    var /= 1
    var %= 1
    echo var
}

let value_a = 4
value(value_a)

let value_b = -1
value(value_b)
```

However... could you modify the body of this function to make better tests? And also echo results after each operation so that we know where a problem might show up. Ex.:

```php
var += 123
echo var

var -= 23
echo var

var *= 4
echo var

var /= 100
echo var

// ...
```

Reply:
If you're going to keep all these tests in one file, I suggest annotating the output, e.g.:
```
var += 123
echo "var += 123 => {var}"
```
Or you could clone the non-reference versions of the tests, as outlined above.

Reply:
Oh yes. I agree

Reply:
Right now this test is failing as it is so before I change as you suggest can you help me understand what is wrong. 
I didn't knew about this `ref` feature.

Reply:
You can learn more about it here: https://docs.amber-lang.com/basic_syntax/functions#variable-references-ref